hppc/src/main/templates/com/carrotsearch/hppc/KTypeVTypeWormMap.java

```diff
@@ -121,12 +121,11 @@ public KTypeVTypeWormMap(KTypeVTypeAssociativeContainer<? extends KType, ? exten
   @Override
   public KTypeVTypeWormMap<KType, VType> clone() {
     try {
-      /* #if ($templateOnly) */
-      @SuppressWarnings("unchecked") /* #end */
+      /* #if ($templateOnly) */ @SuppressWarnings("unchecked") /* #end */
               KTypeVTypeWormMap<KType, VType> cloneMap = (KTypeVTypeWormMap<KType, VType>) super.clone();
-      cloneMap.keys = Arrays.copyOf(keys, keys.length);
```

Comment:
This is an intriguing question - whether clone here is faster than Arrays.copyOf (I wonder if JMH's Alexey Shipilev has a benchmark for this).

Reply:
I changed here to be more consistent with what you did with HashMap. I don't think it really changes the speed. Primitive array cloning seems to be fast and it just requires less code.

Some links I found about the perf comparison:
https://code.i-harness.com/en/q/2c02122
https://www.artima.com/intv/bloch.html#part13 (Josh Bloch)